crates/wasmparser/src/readers/core/operators.rs

```diff
@@ -1176,3 +1187,1280 @@ impl<'a> Iterator for OperatorsIteratorWithOffsets<'a> {
         Some(result)
     }
 }
+
+/// Trait implemented by types that can visit all [`Operator`] variants.
+#[allow(missing_docs)]
+pub trait VisitOperator<'a, Input> {
+    /// The result type of the visitor.
+    type Output: 'a;
+
+    /// Visits the [`Operator`] `op` using the given `input`.
+    ///
+    /// # Note
+    ///
+    /// This is a convenience method that is intended for non-performance
+    /// critical use cases. For performance critical implementations users
+    /// are recommended to directly use the respective `visit` methods or
+    /// implement [`VisitOperator`] on their own.
+    #[inline(always)]
+    #[rustfmt::skip]
```

Comment:
This PR commonly uses `rustfmt::skip`, but could that be removed? It may make code a bit longer but I think having consistent styling is more important myself.

Reply:
The `#[rustfmt::skip]` are only used for parts of bulky code to allow for that part to be easily bulk-transformed. I commonly used this throughout the lifetime of this PR and would only want to remove those annotations once we are absolutely sure to merge it.